cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,188 @@
+    cy.get('div')
+      .should('contain', worker.salary);
+    cy.get('div')
+      .should('contain', worker.department);
+  });
+
+  it('should provide an ability to delete worker', () => {
+    cy.get('#delete-record-1')
+      .click();
+    cy.get('#delete-record-1')
+      .should('not.exist');
+  });
+
+  it('should provide an ability to delete all workers', () => {
+    cy.get('#delete-record-1')
+      .click();
+    cy.get('#delete-record-2')
+      .click();
+    cy.get('#delete-record-3')
+      .click();
+    cy.get('.rt-noData')
+      .should('contain', 'No rows found');
+  });
+
+  // eslint-disable-next-line max-len
+  it('should provide an ability to find worker in search field and edit it', () => {
+    cy.get('#searchBox')
+      .type('Cierra');
+    cy.get('#edit-record-1')
+      .click();
+    cy.get('#registration-form-modal')
+      .should('contain.text', 'Registration Form');
+    cy.get('#firstName')
+      .clear();
+    cy.get('#firstName')
+      .type(worker.firstName);
+    cy.get('#lastName')
+      .clear();
+    cy.get('#lastName')
+      .type(worker.lastName);
+    cy.get('#userEmail')
+      .clear();
+    cy.get('#userEmail')
+      .type(worker.email);
+    cy.get('#age')
+      .clear();
+    cy.get('#age')
+      .type(worker.age);
+    cy.get('#salary')
+      .clear();
+    cy.get('#salary')
+      .type(worker.salary);
+    cy.get('#department')
+      .clear();
+    cy.get('#department')
+      .type(worker.department);
+    cy.get('#submit')
+      .click();
+    cy.get('#searchBox')
+      .clear();
+    cy.get('div')
+      .should('contain', worker.firstName);
+    cy.get('div')
+      .should('contain', worker.lastName);
+    cy.get('div')
+      .should('contain', worker.email);
+    cy.get('div')
+      .should('contain', worker.age);
+    cy.get('div')
+      .should('contain', worker.salary);
+    cy.get('div')
+      .should('contain', worker.department);
+  });
+
+  it('should provide an ability to validate data after creating worker', () => {
+    cy.get('#addNewRecordButton')
+      .click();
+    cy.get('#registration-form-modal')
+      .should('contain.text', 'Registration Form');
+    cy.get('#firstName')
+      .type(worker.firstName);
+    cy.get('#lastName')
+      .type(worker.lastName);
+    cy.get('#userEmail')
+      .type(worker.email);
+    cy.get('#age')
+      .type(worker.age);
+    cy.get('#salary')
+      .type(worker.salary);
+    cy.get('#department')
+      .type(worker.department);
+    cy.get('#submit')
+      .click();
+    cy.get('div').should('contain', worker.firstName);
+    cy.get('div').should('contain', worker.lastName);
+    cy.get('div').should('contain', worker.email);
+    cy.get('div').should('contain', worker.age);
+    cy.get('div').should('contain', worker.salary);
+    cy.get('div').should('contain', worker.department);
+  });
+
+  it.only('should provide an ability to check search by all column val', () => {
```

Comment:
remove 'only' just to make sure that all your tests from test suite are running.